cheddar/cheddar-context/src/test/java/com/clicktravel/cheddar/request/context/DefaultSecurityContextTest.java

```diff
@@ -58,14 +62,34 @@ public void shouldReturnTeamId_withTeamInContext() {
         assertEquals(Optional.of(teamId), returnedTeamId);
     }
 
+    @Test
+    public void shouldReturnIdentityProviderId_withTeamInContext() {
```

Comment:
maybe call this `shouldReturnIdentityProviderId_withIdentityProviderInContext()` ?

Reply:
Thanks, I'll update, I thought this was regarding Team Context in regards to Having an accompanying Team Id Header, Looking at this with the view that this "in context" just means is available in the header map I will update all of the tests to use the same `final String appId = randomBoolean() ? null : randomId();` randomly plopping the other headers in around it, currently identityProviderId is always added to all the other tests.